package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^6.6.0",
-    "ember-getowner-polyfill": "^1.1.1"
+    "ember-getowner-polyfill": "^1.1.0 || ^2.0.0"
```

Comment:
Is it more reliable to leave the option of installing the `1.1.x` version of this addon for older consuming apps?